app/services/onlinetesting/phase3/Phase3TestService.scala

```diff
@@ -286,31 +286,7 @@ trait Phase3TestService extends OnlineTestService with Phase3TestConcern {
       )
     )
   }
-
   //scalastyle:on method.length
-  @deprecated
-  private def registerAndInviteApplicant(application: OnlineTestApplication, emailAddress: String, interviewId: Int, invitationDate: DateTime,
-                                         expirationDate: DateTime
-                                        )(implicit hc: HeaderCarrier, rh: RequestHeader): Future[LaunchpadTest] = {
-    val customCandidateId = "FSCND-" + tokenFactory.generateUUID()
-
-    for {
```

Comment:
Unrelated to the ticket, but i had fixed a couple of warnings, hoping it's ok here